### ISSUE TITLE
The problem of assign frame-level label during training

### ISSUE THREAD

**Issue by buaa-luzhi** [ColumbiaDVMM/CDC#11]
Hello, Zheng

        Thanks for your sharing!
        I am very interested in S-CNN and  CDC published CVPR.
   
        When run  "gen_test_bin_and List" , for multiclass problems, each class labels are 0.
        So, i think this script does not satisfy the requirement of producing training set.
        
        Do you think I am right?   Looking forward to your reply. 

Best regards

**Comment by HYPJUDY** [comment]
Hi, with regard to this issue, there a line in `gen_test_bin_and_list.py`
```
np.zeros(len(imglist)) # during training, assign frame-level label here instead of all zeros
```
I also have a question:
What labels should be assigned to the 22 possible frame-level classes(from the first to the last: background, action1-20, ambiguous)?
1. Should I assign number 1-20 to the 20-action frames or the label provided by THUMOS14 as follows:
[7, 9, 12, 21, 22, 23, 24, 26, 31, 33, 36, 40, 45, 51, 68, 79, 85, 92, 93, 97]
2. What labels should I assign to background and ambiguous frames?

Currently I assign [0, 7, 9, 12, 21, 22, 23, 24, 26, 31, 33, 36, 40, 45, 51, 68, 79, 85, 92, 93, 97, 0] to the 22 possible frame-level classes. Does it make sense?
Thanks!

**Comment by wangwen39** [comment]
Yes, have you solved the problem of asign frame-level label during training? can you share your code here ?
Thank you so much! Looking forward to your replay.

**Comment by HYPJUDY** [comment]
Hi @wangwen1995 , maybe you can gain some enlightenment from #12 .
